src/Storefront/Test/Framework/Routing/StorefrontResponseTest.php

```diff
@@ -0,0 +1,34 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Test\Framework\Routing;
```

Comment:
Please move the test into the new test structure under `/tests/unit/php/Storefront/Framework/Routing`